app/views/content/non-uk-teachers/get-an-international-relocation-payment.md

```diff
@@ -15,7 +15,7 @@ talk_to_us: false
 inset_text:
   applications-open:
     text: |-
-      <p>Applications for the <a href="https://get-a-teacher-relocation-payment.education.gov.uk/"> international relocation payment (IRP)</a> for teachers and salaried trainees are open from 2 January 2024 to 29 February 2024.
+      <p>Applications for the <a href="https://get-a-teacher-relocation-payment.education.gov.uk/"> international relocation payment (IRP)</a> for teachers and salaried trainees are are now closed. They will reopen between 2 April and 31 May 2024. 
```

Comment:
@laurahermionetennant92 not sure this needs to link to the IRP page as people can't currently apply, and there's a link to the page in the How to apply section so we don't need the same link twice on the page? Think it's enough to just give them the info at this point. Otherwise they're being taken away from the page to something they can't currently do.